### PR TITLE
chore: bump MIN_PEER_PROTO_VERSION to 70215

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
simplifies logic, removes branches

This protocol version is from v14 in May of 2019, should be more than safe to bump this in v18

### CODE REVIEW

File: src/version.h [hunk 1]
```diff
@@ -17,30 +17,17 @@ static const int PROTOCOL_VERSION = 70221;
...
-//! introduction of LLMQs
-static const int LLMQS_PROTO_VERSION = 70214;
-
-//! introduction of SENDDSQUEUE
-//! TODO we can remove this in 0.15.0.0
-static const int SENDDSQUEUE_PROTO_VERSION = 70214;
-
 //! minimum peer version accepted by mixing pool
-static const int MIN_COINJOIN_PEER_PROTO_VERSION = 70213;
+static const int MIN_COINJOIN_PEER_PROTO_VERSION = 70215;
```
Comment (UdjinM6) [inline src/version.h:30]: Should drop it 5d777b5a28 instead of bumping it imo

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
> should be more than safe to bump this in v18

this needs confirmation from ios/android mobile teams + see below

**Comment by PastaPastaPasta** [comment]
> > should be more than safe to bump this in v18
> 
> this needs confirmation from ios/android mobile teams + see below

I've already cleared it with android and iOS, android has a small bit of an issue with it, maybe like 1-2% of installs, almost certainly all inactive. iOS has no issue with a bump at all

**Comment by UdjinM6** [comment]
> > > should be more than safe to bump this in v18
> > 
> > 
> > this needs confirmation from ios/android mobile teams + see below
> 
> I've already cleared it with android and iOS, android has a small bit of an issue with it, maybe like 1-2% of installs, almost certainly all inactive. iOS has no issue with a bump at all

ok, cool :) and what about https://github.com/dashpay/dash/pull/4764#discussion_r847691282?

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Pull request merged**